docs/pages/database-access/auto-user-provisioning/mysql.mdx

```diff
@@ -97,6 +97,18 @@ database queries in the Teleport Audit Logs, when the Teleport username is over
 
 ## Step 3/3. Connect to the database
 
+<Admonition type="note" title="Database permissions">
+By default, the newly created won't have permissions to access any particular database. These permissions must be
+granted either through the common role `teleport-auto-user` or through database-specific roles, such as `reader`.
```

Comment:
I would not recommend granting permissions to `teleport-auto-user` as the role is created by `teleport-admin` and the role is only meant for tracking teleport-managed users.

If my understanding is correct, if any of the roles assigned to the auto-provisioned user has some meaningful grant, `tsh db connect` should succeed on the db-name that the role grants? I would either convert this block to a Troubleshooting or a Prerequisites. What do you think?

Reply:
Sure, that works. Although if `teleport-auto-user` shouldn't be touched, then in practice does it mean the user should have at least one other role?

Reply:
Yes. We can expect at least one `db_roles` to be assigned through the Teleport role (or roles). Though seems we are not enforcing this (like not checking against empty `db_roles`)

Reply:
There might be some cases where this makes sense, so perhaps we don't need to change the requirements here. I think the docs are fine now, going to merge in a bit.